src/Components.tsx

```diff
@@ -3,6 +3,43 @@ import {useUpdateWithSetter} from './utils/useUpdateWithSetter';
 import {CollectionOptions, FetchEphemeralKeyFunction} from './types';
 import {NotificationCount} from '@stripe/connect-js';
 
+export const ConnectAppInstall = ({
+  app,
+  onAppInstallStateFetch,
+  onAppInstallStateChange,
+}: {
+  app: string;
+  onAppInstallStateFetch: () => void;
```

Comment:
I think this is missing the actual values to callback right (as in the state of the installed app)?